### PR TITLE
Feature/1 creating basic rule

### PR DESCRIPTION
**Pull request by jasrial**

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,33 +1,3 @@
...
-}
-
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-  to {
-    transform: rotate(360deg);
-  }
+  text-align: left;
```
Comment (vlabs) [inline src/App.css]: entirely remove file

File: src/components/Rule.css [hunk 2]
```diff
@@ -0,0 +1,7 @@
+.Rule {
+  margin: 10px;
+}
+
+.Row {
+  margin-top: 25px;
+}
```
Comment (vlabs) [inline src/components/Rule.css]: FYI try to avoid css on the long term

ok for now

File: src/App.css (hunk 1 above)
Comment (jasrial) [reply src/App.css]: 7d6c56872eca0fc74af704e1e71fdb602661df15

File: src/components/Rule.css (hunk 2 above)
Comment (jasrial) [reply src/components/Rule.css]: ok!

**Comment by vlabs** [comment]
u did not merge it

**Pull request merged**